processor/src/test/java/org/mapstruct/ap/testutil/runner/GeneratedSource.java

```diff
@@ -58,16 +86,35 @@ static void clearCompilingStatement() {
         GeneratedSource.compilingStatement.remove();
     }
 
+
+    /**
+     * Adds more mappers that need to be compared.
+     *
+     * The comparison is done for mappers and the are compared against a Java file that matches the name of the
+     * Mapper that would have been created for the fixture.
+     *
+     * @param fixturesFor the classes that need to be compared with
+     */
+    public void addComparisonToFixtureFor(Class<?>... fixturesFor) {
```

Comment:
You are right. I will do that. Just one question should we use varargs just one argument?